ui/packages/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -138,9 +126,6 @@ module.exports = Marionette.CollectionView.extend(
           if (typeof json === 'undefined') {
             return attributeToVal
           } else {
```

Comment:
✏️ This `else` is not needed. Since you're already refactoring it why not get rid of this 😄 ?